src/ui/pages/WalletSelect/WalletSelect.tsx

```diff
@@ -165,11 +192,13 @@ export function WalletSelect() {
         >
           <WalletList
             walletGroups={walletGroups}
+            walletsMeta={walletsMeta || []}
```

Comment:
Same question here about extending param list for WalletList. May be it is better to make it more abstract and make all wallet-meta related calculations here?

Reply:
totally agree! refactored ✅